Tests/run_tests.py

```diff
@@ -91,8 +91,7 @@ def is_pypy():
     'Bio.Phylo.Applications._Fasttree',
     'Bio.Phylo._io',
     'Bio.Phylo.TreeConstruction',
-    'Bio.Phylo._utils',
-    'BioSQL.BioSeqDatabase',
+    'Bio.Phylo._utils'
```

Comment:
Could you replace the trailing comma on the line above?

This is a minor style point, but the idea here is then adding/removing entries only affects one line in the diff. There is even a flake8 plugin to enforce trailing commas.

Reply:
I've changed it, yeah. Couldn't put my finger on the problem here.